### DISCUSSION TITLE
Range Option for Datepicker

### DISCUSSION THREAD

**Discussion by pmcroroscher** [formkit/formkit#1323] [Ideas]
It would be cool to implement the option to use the pro field datepicker in order to select a range of dates, like so:

<img width="537" alt="Bildschirmfoto 2024-04-22 um 16 02 23" src="https://github.com/formkit/formkit/assets/139750421/c4606d8c-9d0c-41d2-998a-64dd1a797ae7">

**Comment by radusuciu** [comment]
This is a feature I'd love to have as well

**Comment by Tragio** [comment]
Some inspiration for this: https://ui.nuxt.com/components/date-picker#daterangepicker
